### PR TITLE
Linkedin networking

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter -2 created 👍

### CODE REVIEW

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
I like the direction and amount of content on this PR. But networking is one of the thing that really helps to excel on Linkedin. 
The content on the PR - is good but could be found very easily on internet in one search. I would like to read something which I haven't read so far - and I have read a lot of stuff about networking. 
I expect you to do deeper research. Current state of information is very naive. 

With deeper search on this topic - will really help you to gain a massive network on Linkedin. Massive means - 10,000+

**Comment by Vaibhavpratapsingh22** [comment]
@udeysingh95, I searched a lot on networking but everywhere I found the same tactics that I have already mentioned. Though, I have changed some content which I founded more relevant. So please check it 👍

File: 11-linkedin/1-profile.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+You don't need to request a skill endorsement in order to receive one. By default, you'll receive a notification when a connection endorses one of your skills. However, you can manage the frequency of the skill endorsement notifications.
+<p text align="center"><img src="https://i.insider.com/5e91008bdcd88c16237eb6c3?width=1100&format=jpeg&auto=webp"></p>
+
+
+
+##  Accomplishments
+Accomplishments help the viewer to know about you in terms of work that you have done in past years. So in this section you can add the projects that you have completed and what was your position in that project. You can add the languages you know, blogs that you have written, honors and awards that you have achieved along with your studies. 
+<p text align="center"><img src="https://neilpatel.com/wp-content/uploads/2018/03/linkedin-accomplishments.png"></p>
+
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/1-profile.md:42]: Why there is an extra spacing here and below in the next topic also?

File: 11-linkedin/2-networking.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+1. Spend a little time getting to know how the group works, who’s most vocal, what content gets the most buzz, etc.
+
+2. You can introduce yourself. Let other members know you are new to the group and share a little bit about why you are there.   
+
+### Nurture Your Network
+By the word nurture I mean taking care of your relationship with connections. Adding people to your network won’t help you build a relationship with them. You need to engage with them regularly. Here’s how:
+
+**1. Interact**: 
+ 1. Provide status updates on a regular basis. It keeps you visible to the people in your brand community.
+
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/2-networking.md:40]: Extra Spacing in between.

File: 11-linkedin/2-networking.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Try to post on a regular basis and include relevant hashtags on your content which will extend reach beyond your first-degree connections. If you’re job seeking, this could prompt hiring managers or other professionals to reach out to you directly. For more tips to share your insights on LinkedIn, read on here.
+
+### 4. Create Win/Win Situations
+It's very important that both you and the other party gain equally when networking. If you benefit more than the other person, he or she will feel cheated and used. When that happens, you won't be able to go back and built a long-term relationship.
+### 5. Become A Connector
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/2-networking.md:71]: There is space before every new heading and here there is no space.

**Review by SaarthakDeshmukh — CHANGES_REQUESTED** [review]
Networking can be done through campaigns also, Check some more links around that, It might be helpful. I liked the way you have summarized some topics in this PR. Good Job!

**Comment by udeysingh95** [comment]
Networking is heavily done using #opennetworkers - this is a concept on Linkedin - there are a lot of people who are interested in networking to anyone - and these people often post for more networking - and you can connect to hundreds of people through one post only. Read and add to the content about it.

**Review by udeysingh95 — APPROVED** [review]

**Review by SaarthakDeshmukh — APPROVED** [review]

**Pull request merged**